dts/bindings/fuel-gauge/battery_cutoff.yaml

```diff
@@ -0,0 +1,30 @@
+#
+# Copyright 2023 Google LLC
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+description: |
+
+  Properties for fuel-gauges that may control battery cutoff, this is common in SBS compliant or
+  similarly smart battery fuel gauges.
+
+  Note: These properties are to be used with meaningful defaults in fuel gauge ICs that implement
+  battery cutoff support. See the default fuel gauge SBS Gauge compatible as an example.
```

Comment:
Drop the word 'support' ?

What does 'default fuel gauge SBS Gauge compatible' mean? It might be missing hyphens, but I'm not sure

Reply:
I think (hope) I made it more clear in the following push.